XRTK-Core/Packages/com.xrtk.core/Tests/Services/ITestService.cs

```diff
@@ -1,12 +1,33 @@
 ﻿// Copyright (c) XRTK. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System.Collections.Generic;
 using XRTK.Interfaces;
+using XRTK.Interfaces.CameraSystem;
 
 namespace XRTK.Tests.Services
 {
     internal interface ITestService : IMixedRealityService
     {
         bool IsEnabled { get; }
+
+        /// <summary>
+        /// The list of <see cref="IMixedRealityDataProvider"/>s registered and running with the system.
+        /// </summary>
+        IReadOnlyCollection<IMixedRealityDataProvider> DataProviders { get; }
```

Comment:
@FejZa @SimonDarksideJ Thoughts about promoting this to `IMixedRealityService`?

Reply:
In a separate PR, I am going to unweave the RegisteredServiceProviders